### PR TITLE
Recently updated sort

### PR DESCRIPTION
**Pull request by moore3071**
Allow sorting by which crates were most recently updated as mentioned in #1210. The option is called 'recent-updates' in order to match the naming convention of 'recent-downloads'

### CODE REVIEW

**Comment by carols10cents** [comment]
Oh geez I am so sorry to have let this sit for so long. This is actually important for performance reasons for an API user of ours.

@jtgeibel @sgrif could yinz prioritize code review of this please?

**Comment by sgrif** [comment]
Will take care of it when I'm back from RailsConf

**Comment by sgrif** [comment]
Is there a reason we needed to introduce the `time` crate just for this? Could this be done with `std::time::Duration` or `chrono::Duration`?

File: src/tests/krate.rs [hunk 1]
```diff
@@ -282,36 +283,155 @@ fn exact_match_first_on_queries() {
...
-        ::CrateBuilder::new("other_sort", user.id)
+        krate4 = ::CrateBuilder::new("other_sort", user.id)
             .description("other_sort const")
             .downloads(999_999)
             .expect_build(&conn);
+
+        // Set the updated at column for each crate
+        let updated = Utc::now().naive_utc();
+        update(&krate1)
+            .set(crates::updated_at.eq(updated - Duration::weeks(3)))
```
Comment (sgrif) [inline src/tests/krate.rs]: We don't need to do this in Rust. This can just be `now - 3.weeks()` (`use diesel::dsl::*`)

**Comment by carols10cents** [comment]
I did a rebase and a force-push to avoid having to merge and then revert the Cargo.toml changes 😅 Sorry for letting this sit so long!

**Pull request merged**